judgments/views/advanced_search.py

```diff
@@ -12,13 +12,26 @@
     as_integer,
     has_filters,
     paginator,
+    parse_date_parameter,
     perform_advanced_search,
     preprocess_query,
 )
 
 
 def advanced_search(request):
     params = request.GET
+
+    try:
+        from_date = parse_date_parameter(params, "from")
+    except ValueError:
+        from_date = None  # TODO - we need to add a mechanism
```

Comment:
Let's make a trello card - i think it's an 'important enhancement' rather than an essential - since this sits behind  a feature flag it's in a state that's user-test-able (the validation never outright fails with eg a 500, even if it doesn't surface errors), even though if we wanted to *release* it publicly we'd need to think harder about error recovery. Will add a card!